### PR TITLE
Add the "Hasura E2E tests chronicles, February 2023" chapter

### PR DESCRIPTION
**Pull request by NoriSte**
Bring [@NoriSte's article](https://dev.to/noriste/hasura-e2e-tests-chronicles-february-2023-24ki) to the repo.

### CODE REVIEW

File: sections/real-life-examples/hasura-e2e-tests-chronicles-february-2023.md [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+... We opted for the hardest but best decision: to eradicate the E2E tests!
+
+
+![The description of the big PR removing the E2E tests](/assets/images/hasura-chronicles/wipeout-pr.png)
+_The description of the big PR removing the E2E tests._
+
+
+Let's start with a clean slate and think about the future! Some points that will positively impact the E2E tests:
+
+1.  We will consider **Playwright instead of Cypress** due to the speed and fewer resources required. The first round of using it went well. Now we need to check if it fulfills all our needs.
```
Comment (muratkeremozcan) [inline sections/real-life-examples/hasura-e2e-tests-chronicles-february-2023.md:256]: There are many problems, not related to the framework, but another framework is being considered.

If you have to hard wait in a test for a slow API, that test is going to be slow anywhere. 

I am curious how this went.

**Review by muratkeremozcan — COMMENTED** [review]
Sounds like Hasura doesn't have test architect, or a  team of testers. 

Can you summarize the layman's takeaways at the end? It's a good story, and what are the lessons the readers should learn from this, and not repeat your mistakes.

File: sections/real-life-examples/hasura-e2e-tests-chronicles-february-2023.md (hunk 1 above)
Comment (NoriSte) [reply sections/real-life-examples/hasura-e2e-tests-chronicles-february-2023.md:256]: It went nowhere since I no work there and, from what I know, they restarted writing e2e tests the old way and they have the same exact problems again...

**Comment by NoriSte** [comment]
> Sounds like Hasura doesn't have test architect, or a team of testers.
> 
> Can you summarize the layman's takeaways at the end? It's a good story, and what are the lessons the readers should learn from this, and not repeat your mistakes.

I think the takeaways are
1. Treat the e2e tests as a product and care about them. If they add friction or you do not have time to care about them, it's better not to have them at all
2. Never hide the dust under the carpet with e2e tests, things can only get worse

WDYT?
